### DISCUSSION TITLE
[/lastonline] Little summery of x in the last y

### DISCUSSION THREAD

**Discussion by mrfloris** [Zrips/CMI#7737] [Enhancement]
Hey zrips, thanks for considering this, hopefully it's possible to do.

`/cmi lastonline` shows unique player joins in the last x period of time. It would be kinda cool if it could start with a bit of mini stats for the last x per hour, day, week, month, year? With a config.yml as to which ones to show (or not).

```
-----------------------------
Stats: 12 in the last hour, 123 in the last 24 hours,
Stats: 1,230 in the last week, 5,441 in the last month.
-----------------------------
1. notch
2. zrips
3. mrfloris
4. whatever
```
Something like that?

**Comment by Zrips** [comment]
Sounds like a plan

**Comment by mrfloris** [comment]
Awesome, thank you.
